test/client/vendor/async.js

```diff
@@ -0,0 +1,709 @@
+/*! async.js | https://github.com/caolan/async/blob/master/LICENSE */
+/*global setTimeout: false, console: false */
+(function () {
+
+    var async = {};
```

Comment:
We need to get rid of async in the tests in favor of promises. Let's create milestone v1.0 issue for this task.